### ISSUE TITLE
Bot remains as a background process after closing Composer 

### ISSUE THREAD

**Issue by BruceHaley** [microsoft/BotFramework-Composer#9059]
## Describe the bug

When I close Composer, the bot I was working with remains as a Windows background process. 

I would expect all Windows processes from Composer to be terminated when Composer is closed.

## Repro

1. Open Composer (I am using v 2.1.2).
2. Click on a bot in Composer to open it for editing.
3. Click Start bot.
4. Click Open web chat.
5. Close Composer.
6. Open Windows Task Manager.
7. Search the list of processes for one bearing the name of the bot opened in step 2.

Expected: The process would not be found, as it would terminate when Composer was closed.
Actual: The process is listed.

**Comment by sw-joelmut** [comment]
Hi @OEvgeny, we've been working on this issue and created the PR #9536 with a possible fix. Let us know if these changes are ok with you.
**TL;DR:** the electron server was closing the client before it finished executing the cleanup process.

**Comment by OEvgeny** [comment]
@sw-joelmut While I was working on this one, I found a couple of other issues, I will work to merge the two PRs together to fix the issue on all platforms.

**Comment by OEvgeny** [comment]
@sw-joelmut updated mine PR with both changes. Could you take a look?
